lib/adal-aad-node/src/i18n.ts/defaultI18n.ts

```diff
@@ -0,0 +1,24 @@
+import { StringLookup, InteractionRequiredContext } from "../models";
```

Comment:
What is this file for? are we getting rid of src/errors/errors.ts?

Reply:
This is for normal string lookup, rather than error lookup. Might eventually combine them into one.

Reply:
Don't want to cause merge conflicts right now.